### ISSUE TITLE
peerDependency @swc/core isn't optional

### ISSUE THREAD

**Issue by JiPaix** [axe-me/vite-plugin-node#36]
```ts
import { join } from 'path';
import { defineConfig } from 'vite';
import { VitePluginNode } from 'vite-plugin-node';

export default defineConfig({
  server: {
    port: 3000,
  },
   plugins: [
    ...VitePluginNode({
      adapter: 'express',
      appPath: './index.ts',
      exportName: 'viteNodeApp',
      tsCompiler: 'esbuild',
      swcOptions: {},
    }),
  ],
});
```

```console
error during build:
Error: Cannot find module '@swc/core'
```

could you temporarily fix it by changing this line until it's fixed?
https://github.com/axe-me/vite-plugin-node/blob/809f465ae191377158c8f72f55e7bcbc9a03cf7a/package.json#L44

**Comment by RDIL** [comment]
Well but it is optional... for instance if you don't use it, you shouldn't have to install it.

**Comment by axe-me** [comment]
fixed in `1.0.0` by lazy loading the module
